_data/package_index.yml

```
@@ -325,6 +325,13 @@
   tags: interface-generator python
   license: GPL-3.0
   version: none
+  
+- name: convert_FORTRAN_case
+  github:  dav05/convert_FORTRAN_case
+  description: case conversion of files writen in fixed form fortran
+  categories: programming
+  tags: formatter converter
+  license: GPL-3.0  
```

This is autodetected, but since it has no release, we should disable the version info:
```suggestion
  version: none  
```